assets/js/pages/detail.js

```diff
@@ -440,9 +516,48 @@ module.exports = function (name) {
         publish.displayQueueList();
     }
 
-    return {
-        init: init
+    /**
+     * Reset parameters
+     */
+    function resetParams() {
+        // detail = {};
+        detail.article = [];
+        detail.errors = [];
+        detail.detailEvents = [];
+        detail.detailArticle = [];
+        detail.scheduleStatus = [];
+        detail.queryParams = {};
     }
 
-
-}();
+    var det = {
```

Comment:
Could this be given a better name? Coming across references to it elsewhere in the file, it's not clear what it is.